### ISSUE TITLE
Feature request: Add option to reset the home position only the first time the craft is armed

### ISSUE THREAD

**Issue by Dr-Drone** [iNavFlight/inav#3143]
I was thinking if it was possible to not have the home point reset when disarmed and have the home point saved only when powered on. This may help save the aircraft if at all it decides or we accidentally disarm in flight.

Another feature would be to have iNAV return to home maintaining the current altitude and once it reaches home it could then lower it's altitude.

**Comment by stronnag** [comment]
I'm sure we discussed this before (can't recall where) and came to the conclusion that the best solution would a setting along the lines of:
```
set  inav_reset_home =  FIRST_ARM
Allowed values: NEVER, FIRST_ARM, EACH_ARM
# just like inav_reset_altitude
```
Or maybe one `inav_reset_important_stuff` setting that does both.

**Comment by Dr-Drone** [comment]
Yeah that seems just about right.

**Comment by danarrib** [comment]
This is a VERY important feature. Anyone's planning to do it?

**Comment by OlivierC-FR** [comment]
I second this suggestion !
Mid-air disarms are annoying enoughnalready, but then you also lose the direction and distance to home, and that's not helping at all.

**Comment by danarrib** [comment]
Awesome! Looking forward to test it!

**Comment by Dr-Drone** [comment]
Nice! 👏

**Comment by fiam** [comment]
Fixed in #3143
